deurdu/README.md

```diff
@@ -1,3 +1,22 @@
 This directory contains all data and source code for deurdu blogging
 website.
 
+#For frontend or client side - to make it activate
+1) First install node.js in system
+Goto --> 'https://nodejs.org/en/download/source-code'
+
+2) Make sure to have XAMPP CONTROL PANEL
+
+3) After opeing the 'deurdu' folder
+    move to client using 'cd client'
+    type 'npm i' -- this will install all the packages with a single command
+    then type 'npm run dev' -- this will start project on localhost:5173
```

Comment:
After installing `npm` on my Ubuntu machine, and running these steps, I ran into
```bash
npm run dev --

> client@0.0.0 dev /home/q550356/sample_code/deurdu/client
> vite

/home/q550356/sample_code/deurdu/client/node_modules/vite/bin/vite.js:2
import { performance } from 'node:perf_hooks'
       ^

SyntaxError: Unexpected token {
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! client@0.0.0 dev: `vite`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the client@0.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/q550356/.npm/_logs/2024-09-29T17_33_37_538Z-debug.log
```

Reply:
Haven't used any of the **performance** named react hook's, will debug it on live call.

Reply:
Looks like a newer version of `node` as well as `npm` was needed: https://askubuntu.com/questions/426750/how-can-i-update-my-nodejs-to-the-latest-version